HACKING.rst

```diff
@@ -0,0 +1,32 @@
+NAPALM Style Commandments
+=======================
+
+- Step 1: Read the OpenStack Hacking Standards
+  `here <http://docs.openstack.org/developer/hacking/>`_.
+- Step 2: Read on
+
+NAPALM Specific Commandments
+--------------------------
+
+- [N319] Validate that debug level logs are not translated
+- [N320] Validate that LOG messages, except debug ones, have translations
+- [N321] Validate that jsonutils module is used instead of json
```

Comment:
This is another openstack-y thing. I am not aware of the value that the `jsonutils` module in `oslo` brings over json, so this can probably be removed.